src/ReCaptcha.php

```diff
@@ -135,7 +123,7 @@ public function __construct(
             $this->setOptions($options);
         }
 
-        $this->setHttpClient($httpClient ?: new HttpClient);
+        $this->setHttpClient($httpClient ?  : new HttpClient());
```

Comment:
No space between `?` and `:` here, please; this should be the shortened ternary form for "assign default value only if test value is not truthy".